react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -55,19 +55,21 @@ class Game extends Component {
 
     let status = decideGameStatus(winner,this.state.xIsNext);
     return (
-    <div className={styles.game}>
-      <div className={styles.gameBoard}>
-        <Board
-          squares={current.squares}
-          onClick={(i) => this.handleClick(i)}
-         />
-      </div>
-      <div className={styles.gameInfo}>
-        <div>{status}</div>
-        <ol>{moves}</ol>
-      </div>
-      <TableMatches />
-    </div>
+      <>
```

Comment:
Just to know, this isn't supported by some tools versions.
https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#support-for-fragment-syntax